grease-cli/tests/Main.hs

```diff
@@ -457,52 +463,50 @@ makeSanityTests arch d =
         BatchTimeout -> pure ()
         _ -> T.U.assertFailure "Timeout not exceeded"
 
-llvmTests :: T.TestTree
-llvmTests =
-  T.testGroup "LLVM"
-    [ testCase "abort" "abort.llvm.cbl" $ assertSpecificBug Bug.MustFail Nothing
-    , testCase "both branches abort" "both-branches-abort.llvm.cbl" $ assertSpecificBug Bug.OneMustFail Nothing
-    , testCase "double free" "double-free.llvm.cbl" $ assertSpecificBug Bug.MustFail (Just UB.DoubleFree)
-    , testCase "free non pointer" "free-non-ptr.llvm.cbl" $ assertSpecificBug Bug.MustFail (Just UB.FreeUnallocated)
-    , testCase "free stack pointer" "free-stack.llvm.cbl" $ assertSpecificBug Bug.MustFail (Just UB.FreeUnallocated)
-    , testCase "uninit stack" "uninit-stack.llvm.cbl" $ assertSpecificBug Bug.UninitStackRead Nothing
-    , testCase "empty" "empty.llvm.cbl" assertSuccess
-    , testCase "exit" "exit.llvm.cbl" assertSuccess
-    , testCase "function call" "id-bool.llvm.cbl" assertSuccess
-    , testCase "free" "free.llvm.cbl" assertSuccess
-    , testCase "func-ptr" "func-ptr.llvm.cbl" assertSuccess
-    , testCase "func-ptr-error" "func-ptr-error.llvm.cbl" assertCouldNotInfer
-    , testCase "load" "load.llvm.cbl" assertSuccess
-    , testCase "malloc" "malloc.llvm.cbl" assertSuccess
-    , testCase "memcpy" "memcpy.llvm.cbl" assertSuccess
-    , testCase "memset" "memset.llvm.cbl" assertSuccess
-    , testCase "memset large const len" "memset-large-const-len.llvm.cbl" assertSuccess
-    , testCase "null read" "null-read.llvm.cbl" $ assertSpecificBug Bug.MustFail Nothing
-    , testCase "null write" "null-write.llvm.cbl" $ assertSpecificBug Bug.MustFail Nothing
-    , testCase "pointer add offset" "ptr-add-offset.llvm.cbl" assertSuccess
-    , testCase "pointer add large offset" "ptr-add-large-offset.llvm.cbl" assertSuccess
-    , testCase "skip non-void call" "skip-ptr.llvm.cbl" assertSuccess
-    , testCase "skip void call" "skip-void.llvm.cbl" assertSuccess
-    , testCase "store" "store.llvm.cbl" assertSuccess
-    , testCase "trap" "trap.llvm.cbl" $ assertSpecificBug Bug.MustFail Nothing
-    , testCase "user override" "user-override.llvm.cbl" assertSuccess
-    , testCase "user override defun" "user-override-defun.llvm.cbl" assertSuccess
-    , testCase "struct override" "struct-override-caller.llvm.cbl" assertSuccess
-    , testCase "declare in override" "declare-in-override/declare-in-override.llvm.cbl" assertSuccess
-    , testCase "startup override" "startup-override/test.llvm.cbl" assertSuccess
-    , testCase "Rust enum" "rust-enum.llvm.cbl" assertSuccess
-    ]
+llvmTests :: IO T.TestTree
+llvmTests = do
```

Comment:
Do you think it is worth applying the same FileCheck treatment to the Macaw-based S-expression tests? (No need to do so as part of this PR, but it seems like a logical next step.)

Reply:
Yes, definitely. We should probably do it for the tests involving bitcode and binaries too.

https://github.com/GaloisInc/grease/issues/79
#80 
#81